client/components/password-change/password-change.js

```diff
@@ -0,0 +1,75 @@
+import "./index.css";
+import React, {useState} from "react";
+import axios from "axios";
+import qs from "qs";
+import PropTypes from "prop-types";
+
+import {passwordChangeApiUrl} from "../../constants";
+
+const PasswordChangeComponent = ({orgSlug}) => {
+  const [newPassword1, changeNewPassword1] = useState();
+  const [newPassword2, changeNewPassword2] = useState();
+
+  const handleSubmit = () => {
+    const url = passwordChangeApiUrl.replace("{orgSlug}", orgSlug);
+    return axios({
+      method: "post",
+      headers: {
+        "content-type": "application/x-www-form-urlencoded",
+      },
+      url,
+      data: qs.stringify({
+        newPassword1,
+        newPassword2,
+      }),
+    }).catch(error => {
+      console.log(error);
+    });
+  };
+
+  return (
+    <div className="owisp-passchange-container">
+      <div className="title">Change your password</div>
+      <input
+        style={{marginTop: "1rem"}}
+        type="password"
+        name="newPassword1"
+        placeholder="New Password"
+        onChange={e => changeNewPassword1(e.target.value)}
+      />
+      <input
+        type="password"
+        name="newPassword2"
+        placeholder="Repeat Password"
+        onChange={e => changeNewPassword2(e.target.value)}
+      />
+      <div
+        className="owisp-passchange-submit"
+        tabIndex="0"
+        role="button"
+        onClick={() => handleSubmit(newPassword1, newPassword2)}
+        onKeyDown={() => handleSubmit(newPassword1, newPassword2)}
+      >
+        Submit
+      </div>
+    </div>
+  );
+};
+
+const PasswordChangePage = ({orgSlug}) => {
```

Comment:
@someshkar  please merge both the components, there is no need to define 2 components here.